utils/forest_shim/src/randomness.rs

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use fvm_shared::randomness::Randomness as Randomness_v2;
+use fvm_shared3::randomness::Randomness as Randomness_v3;
+use serde::{Deserialize, Serialize};
+use std::ops::{Deref, DerefMut};
+
+/// Represents a shim over Randomness from fvm_shared with convenience methods to convert to an older version of the type
+///
+/// # Examples
+/// ```
+///
+/// // Create FVM2 Randomness normally
+/// let fvm2_rand = fvm_shared::randomness::Randomness(vec![]);
+///
+/// // Create a correspndoning FVM3 Randomness
+/// let fvm3_rand = fvm_shared3::randomness::Randomness(vec![]);
+///
+/// // Create a shim Randomness, ensure conversions are correct
+/// let rand_shim = forest_shim::randomness::Randomness::new(vec![]);
+/// assert_eq!(fvm3_rand, *rand_shim);
+/// assert_eq!(fvm2_rand, rand_shim.into());
+///
+/// ```
+#[derive(PartialEq, Eq, Default, Clone, Debug, Deserialize, Serialize)]
+#[serde(transparent)]
+pub struct Randomness(pub Randomness_v3);
```

Comment:
the inner pub can be removed, we intend only allow interaction with the shim instead

Reply:
fixed